EIPS/eip-2400.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: 2400
+title: Transaction Receipt URI
+author: Ricardo Guilherme Schmidt (3esmit.stateofus.eth), Eric Dvorsak (yenda.stateofus.eth)
```

Comment:
These ENS names will not allow you to use the auto-merge feature. I suggest to use an email or github username if you want to utilize the auto-merge for draft updates.

Reply:
Ok, I will change this back to ENS Usernames once it's move out from Draft.